crm_lead_currency/views/crm_lead_views.xml

```diff
@@ -6,14 +6,28 @@
         <field name="model">crm.lead</field>
         <field name="inherit_id" ref="crm.crm_lead_view_form" />
         <field name="arch" type="xml">
-            <field name="partner_id" position="before">
+            <xpath
+                expr="//group[@name='lead_partner']//field[@name='partner_id']"
+                position="before"
+            >
                 <field name="customer_currency_id" />
                 <field name="is_same_currency" invisible="1" />
                 <field
                     name="amount_customer_currency"
                     attrs="{'invisible': [('is_same_currency', '=', True)]}"
                 />
-            </field>
+            </xpath>
+            <xpath
+                expr="//group[@name='opportunity_partner']//field[@name='partner_id']"
+                position="before"
+            >
+                <field name="customer_currency_id" />
```

Comment:
maybe readonly here to keep the previous behaviour

what do you think @luisg123v ?

Reply:
@suniagajose what do you mean by previous behavior?

If you need it readonly, then it should be set by model so it's applied on every place the field is used

Reply:
Then I agree it should be readonly when it's an opportunity. However, that may only be achieved by model, by using the `states` attribute.

Reply:
@luisg123v @Josesosa07 

sorry, i messed up in my explanation, I meant that in Lead form in 12.0 it is not visible, so in 14.0 it should not be visible either.

12.0
![Design Software Info - Odoo](https://user-images.githubusercontent.com/5335402/236944385-11b19a4e-c36c-468c-b068-f02d14eb5bfe.png)


14.0
![Design Software Info - Odoo (1)](https://user-images.githubusercontent.com/5335402/236944673-74edab7a-019f-4425-addd-eab8b807bff3.png)


then, if a customer request that it should be visible in Lead form this should be perfomed in customer app, not here!